src/meta/processors/admin/InstallPluginProcessor.cpp

```diff
@@ -0,0 +1,67 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "meta/processors/admin/InstallPluginProcessor.h"
+#include "common/fs/FileUtils.h"
+
+namespace nebula {
+namespace meta {
+
+// When installing, just register it, do not open it. open it when using
+void InstallPluginProcessor::process(const cpp2::InstallPluginReq& req) {
+    const auto& pluginItem = req.get_item();
+    auto& pluginName = pluginItem.get_plugin_name();
+    auto& soName = pluginItem.get_so_name();
+    folly::SharedMutex::WriteHolder wHolder(LockUtils::pluginLock());
+
+    // Check if file exists
+    using fs::FileUtils;
```

Comment:
IMO we'd better put the so files in the install dir of nebula.

Reply:
In order to put multiple so files, we currently put it in the share directory of the installation directory